04-sample-workflows/04-02_architectural-workflows/04-02-01_building-mass-generator.md

```diff
@@ -50,21 +50,21 @@ With the geometry of the building generated, the script then evaluates the desig
 
 When geometry is created in Dynamo, often other geometry is needed to facilitate the overall process. 
 
-Please note that all unnecessary geometry has been switched off in Dynamo - this is to ensure the geometry displayed shows the final geometric output.. Any nodes with the preview switched off will not display the output visually in Explore Outcomes.
+Please note that all unnecessary geometry has been switched off in Dynamo - this is to ensure the geometry displayed shows the final geometric output. Any nodes with the preview switched off will not display the output visually in Explore Outcomes.
 
 In this case, only the site boundary \(the tower's external walls and floors\) will be visible. This will provide the user with the ability to critique the design options based on aesthetics within the thumbnail or detail view. 
 
 A series of context buildings have been included in the Revit sample file for a better understanding of the exercise.
 
 ## Benefit of Using Generative Design
 
-On running the script, Dynamo is able to generate one design option based on the user inputs. However, by utilizing Randomize \(?\) as the solver, many more options can be generated in a single run. 
+On running the script, Dynamo is able to generate one design option based on the user inputs. However, by utilizing Randomize for generation method, many more options can be generated in a single run. 
```

Comment:
uh?  are you saying it generates one option because you are maximizing both floor area and surface area so there is only one answer?

Reply:
I do not think so, I just reword the method. @radumg might be able to answer this.

Reply:
Think it was more of a Dynamo vs Refinery comparison, a rewording suggestion below to make this clearer :

```suggestion
When running the script in Dynamo, a single design option is generated for each run, based on the current user inputs. However, by running the script in Generative Design for Revit and selecting Randomize as the generation method, many more options can be generated in a single run. 
```